hw14-warMvc/src/main/java/ru/otus/WebConfig.java

```diff
@@ -0,0 +1,69 @@
+package ru.otus;
+
+import org.hibernate.SessionFactory;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
+import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import org.thymeleaf.spring5.SpringTemplateEngine;
+import org.thymeleaf.spring5.templateresolver.SpringResourceTemplateResolver;
+import org.thymeleaf.spring5.view.ThymeleafViewResolver;
+import org.thymeleaf.templatemode.TemplateMode;
+import ru.otus.core.model.AddressDataSet;
+import ru.otus.core.model.PhoneDataSet;
+import ru.otus.core.model.User;
+import ru.otus.flyway.MigrationsExecutor;
+import ru.otus.flyway.MigrationsExecutorFlyway;
+import ru.otus.hibernate.HibernateUtils;
+
+@PropertySource("classpath:application.properties")
```

Comment:
Файла `application.properties` нет в репозитории, вылетает с ошибкой 

```
org.springframework.beans.factory.BeanDefinitionStoreException: Failed to parse configuration class [ru.otus.WebConfig]; nested exception is java.io.FileNotFoundException: class path resource [application.properties] cannot be opened because it does not exist
...
```